RecoTracker/LSTCore/src/alpaka/TrackCandidate.h

```diff
@@ -128,8 +128,8 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
         unsigned int nPixelQuintuplets = pixelQuintuplets.nPixelQuintuplets();
         for (unsigned int pixelQuintupletIndex : cms::alpakatools::uniform_elements_x(acc, nPixelQuintuplets)) {
           unsigned int pLS_jx = pixelQuintuplets.pixelSegmentIndices()[pixelQuintupletIndex];
-          float eta2 = segmentsPixel.eta()[pLS_jx - prefix];
-          float phi2 = segmentsPixel.phi()[pLS_jx - prefix];
+          float eta2 = pixelSegments.eta()[pLS_jx - prefix];
+          float phi2 = pixelSegments.phi()[pLS_jx - prefix];
           float dEta = alpaka::math::abs(acc, (eta1 - eta2));
           float dPhi = calculate_dPhi(phi1, phi2);
```

Comment:
answering my question about connection to #146  : this will be a conflicting line; and there is another one later.
Just as a heads up to @VourMa